src/Form/ClientType.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Client;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\NotNull;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ClientType extends AbstractType
+{
+    public function __construct(
+        private readonly TranslatorInterface $translator,
+    ) {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name', TextType::class, [
+                'label' => 'create_client_form.client_name.label',
+                'label_attr' => ['class' => 'label'],
+                'constraints' => [
+                    new NotNull(['groups' => 'base']),
+                ],
+                'attr' => [
+                    'class' => 'form-element',
+                ],
+                'help_attr' => [
+                    'class' => 'form-help',
+                ],
+                'help' => 'create_client_form.client_name.help',
+                'required' => true,
+                'row_attr' => ['class' => 'form-element-wrapper'],
+            ])
+            ->add('contact', TextType::class, [
+                'label' => 'create_client_form.client_contact.label',
+                'label_attr' => ['class' => 'label'],
+                'constraints' => [
+                    new NotNull(['groups' => 'base']),
+                ],
+                'attr' => [
+                    'class' => 'form-element',
+                ],
+                'help_attr' => [
+                    'class' => 'form-help',
+                ],
+                'help' => 'create_client_form.client_contact.help',
+                'required' => true,
+                'row_attr' => ['class' => 'form-element-wrapper'],
+            ])
+            ->add('standardPrice', TextType::class, [
```

Comment:
This should be a NumberType: https://symfony.com/doc/current/reference/forms/types/number.html